Lib/importlib/_bootstrap_external.py

```diff
@@ -1033,6 +1039,21 @@ def set_data(self, path, data, *, _mode=0o666):
                                         exc)
 
 
+def _exclude_directory_from_backups(directory):
+    """Exclude the directory from backups using a CACHEDIR.TAG file.
+
+    If the file exists or there's a permission error this is a no-op.
+    """
+    try:
+        with _io.FileIO(_path_join(directory, 'CACHEDIR.TAG'), 'x') as f:
```

Comment:
Why not use `open` here?

Reply:
If I understand correctly `open` is not in scope here as it's a kind of special module that gets selected few names injected into its namespace.